### PR TITLE
Docs Refactoring

### PR DESCRIPTION
**Pull request by gtrkiller**
This PR aims to refactor the docs according to the Documentation Team's defined criteria.

### CODE REVIEW

**Review by merkata — DISMISSED** [review]

**Review by jdkandersson — APPROVED** [review]

**Review by yanksyoon — APPROVED** [review]
LGTM

File: docs/how-to/cache-content-with-openstack-swift.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Content-cache with Openstack/Swift storage
+# Cache content with openstack/swift storage
```
Comment (arturo-seijas) [inline docs/how-to/cache-content-with-openstack-swift.md:1]: Same here
s/openstack/swift/OpenStack/Swift

File: docs/tutorial/getting-started.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Note: `ingress-proxy` is the name of the relation. You can run `juju info conten
 
 The deployment finishes when the status shows "Active".
 
-### Relate to Ingress by using NGINX Ingress Integrator
+### Relate to ingress by using nginx ingress integrator
```
Comment (arturo-seijas) [inline docs/tutorial/getting-started.md:66]: Same here
s/nginx/NGINX

**Review by arturo-seijas — CHANGES_REQUESTED** [review]
Left some comments inline

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by jdkandersson — APPROVED** [review]

**Pull request merged**